trusted-committer/07-becoming-a-trusted-committer.asciidoc

```diff
@@ -1,68 +1,45 @@
+product owners and with management.
+
+In the same vein, they must have shown the willingness and patience to use their skills and spend intentional time upleveling contributors.
+Finally, fulfilling the Trusted Committer role requires a certain emotional maturity to deal with stressful social situations, which are
+bound to come up from time to time. 
+Contributors who satisfy these criteria will be good potential Trusted Committers, in our opinion.
+
+The Trusted Committer role might not appear all that attractive to some contributors as it means spending less time coding. 
+Being nominated for the Trusted Committer role might even be perceived by some as a demotion or as negative feedback on their coding skills. 
+But the opposite is true. 
+Being nominated for the Trusted Committer role usually means someone has recognized your valuable contributions and sees in you the potential to grow and lead. 
+The Trusted Committer role will give you more influence over the evolution of the codebase and will ultimately make you a more complete
+developer. 
+Explaining to contributors how the software works more often than not leads to new insights on part of the Trusted Committer and will help to identify opportunities to improve the software.
+
+Whether you have one or multiple Trusted Committers depends on the size and the risk associated with the software developed by the InnerSource community. 
+The Trusted Committer role is time consuming, and not everyone is willing or able to make that type of commitment. 
+Because of this, some companies use a _Trusted Committer rotation_ system where multiple Trusted Committers share the workload of the Trusted Committer role, and the Trusted Committers who are not _on duty_ can exclusively focus on tech-oriented work. 
+Having more than one Trusted Committer also makes it easier when someone leaves the company or moves on from the role to do something else.
+
+In summary, the Trusted Committer role has to be earned by making valuable contributions - both technical and social - for the benefit of the community. 
+In a healthy community, you will have fellow Trusted Committers at your side. 
+As a Trusted Committer, you will have less time to code, but by acting as a force multiplier you will ultimately be able to boost your value contribution to the community and accelerate your own growth.
 
-The process of officially becoming a Trusted Committer differs from community to community,
-depends on where you are in your InnerSource journey and might evolve over
-time. In grass-roots type communities, the founders often automatically assume
-the role of the Trusted Committer as well. As a community grows, the community or the
-existing Trusted Committers might nominate a https://innersourcecommons.org/resources/learningpath/contributor/index[_contributor_] to become Trusted Committer, which might or might
```

Comment:
Keep all of the links pointing at `innersourcecommons.org`.

Reply:
Fixed this by means of commit suggestions, hence resolving this discussion.